cdc/owner/scheduler.go

```diff
@@ -0,0 +1,358 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package owner
+
+import (
+	"math"
+
+	"github.com/pingcap/failpoint"
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/cdc/model"
+	"go.uber.org/zap"
+)
+
+type schedulerJobType string
+
+const (
+	schedulerJobTypeAddTable    schedulerJobType = "ADD"
+	schedulerJobTypeRemoveTable schedulerJobType = "REMOVE"
```

Comment:
Do we still use `remove` + `add` to represent a table scheduler, is it better to add more scheduling states?